students/BrianNisonger/session4/dict_lab.py

```diff
@@ -0,0 +1,54 @@
+def dictionary_1(food_dict):
+    print(food_dict)
+    del food_preferences["cake"]
+    print(food_preferences)
+    food_preferences["fruit"]="Mango"
+    print(food_preferences)
+    for items in food_preferences:
+        print (items)
+    for items in food_preferences:    
+        print (food_preferences[items])
+    print(bool("cake" in food_preferences))
+    print(bool("Mango" in food_preferences.values()))
+
+def dictionary_2(food_preferences):
+      count_t={}
+      for items in food_preferences:
```

Comment:
careful about names: "items" makes think it's more than one item, when it's only one item in this case.